src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/config/Config.java

```diff
@@ -404,6 +442,44 @@ public void setValues(JSONObject formData) {
         replicationConfig = ReplicationConfig.createReplicationConfigFromJSON(formData);
     }
 
+    private void initializeGerritCommands() {
+        List<String> stringLabels = new ArrayList<>(Arrays.asList("verified", "code-review"));
```

Comment:
`Arrays.asList` already returns `List<String>`, `new ArrayList(..)` is not needed

Reply:
Done

Reply:
Arrays.asList return an immutable list, making it impossible to add new labels to that list, so I kept the original implementation